lib/TTT/game.ex

```diff
@@ -44,12 +56,29 @@ defmodule Game do
   end
 
   def human_move(state, deps) do
-    deps.io.retrieve(Messages.get_message(:choose))
+    deps.io.retrieve(state.current_player <> Messages.get_message(:choose))
     |> Validation.sanitized_move()
     |> Player.analyze(state, deps)
   end
 
-  def computer_move(state) do
+  def hard_computer_move(state) do
+    move =
+      try do
+        Minimax.minimax(state).position
```

Comment:
It feels like this could be merged—the rescue and the case since it comes right after each other.